main.go

```diff
@@ -9,6 +10,7 @@ import (
 func main() {
 	app := drand.CLI()
 	if err := app.Run(os.Args); err != nil {
+		fmt.Printf("%v\n", err)
```

Comment:
This change will cause the error to be printed twice, it gets printed by the error handler here: https://github.com/drand/drand/blob/master/cmd/drand-cli/cli.go#L423-L427

Reply:
🤔 it doesn't always get printed twice though...  for instance, try `drand share`. you don't see the error that you're missing the required flags otherwise. do you have a command that causes duplicate error printing?

Reply:
On your branch:

![Screenshot 2020-07-13 at 15 22 24](https://user-images.githubusercontent.com/152863/87315742-b08ce000-c51c-11ea-8327-2114b2930330.png)